backend/api/prisonApi.ts

```diff
@@ -280,32 +280,6 @@ export const prisonApiFactory = (client) => {
 
```

Comment:
I didn't update `getAdjudicationFindingTypes` (one line above) as that is not something we can source from AdjudicationsAPI

Reply:
But we will need to - I think its an enum - probably there is a need for exposing reference data from the API